src/main/java/uk/gov/hmcts/ccd/domain/service/callbacks/CallbackService.java

```diff
@@ -145,6 +150,10 @@ private <T> Optional<ResponseEntity<T>> sendRequest(final String url,
             if (logCallbackDetails(url)) {
                 LOG.info("Callback {} response received: {}", url, responseEntity);
             }
+
+            final HttpHeaders httpHeadersNew = new HttpHeaders();
+            httpHeadersNew.addAll(responseEntity.getHeaders());
+            storePassThroughHeadersAsRequestAttributes(httpHeadersNew, request);
```

Comment:
A bit less ambiguous!
```suggestion
            storePassThroughHeadersAsRequestAttributes(responseEntity.getHeaders(), request);
```